frontend/src/components/pages/TeamDashboard/Presentation.js

```diff
@@ -33,11 +33,14 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ReviewLineChart = ({ data, minimum, maximum }) => {
-  // TODO calculate maximum and minimum so that all the graphs have the same proportions
-
+const ReviewLineChart = ({ data, allUsersData }) => {
+  const numbersArr = [];
```

Comment:
please calculate the minimum and maximum value elsewhere and then pass those values in as props.

Otherwise for a team of 20 people you will run the calculation 20 times with the same input and the same output.